config/initializers/devise.rb

```diff
@@ -137,7 +137,7 @@ def devise_secret_key_not_set_in_production?
   # their account can't be confirmed with the token any more.
   # Default is nil, meaning there is no restriction on how long a user can take
   # before confirming their account.
-   config.confirm_within = 3.days
+  config.confirm_within = 7.days
```

Comment:
No, in fact I moved to RubyMine and this change came with pulled master or develop made by you. I wanted to undo that to make this return to the 'original' state - I guess something went the other way round. I'll try to rebase this commit to delete this change.